src/validator.js

```diff
@@ -0,0 +1,77 @@
+const { get, isNil } = require("lodash");
+const buildRoutingGraph = require("../src/helpers/build-routing-graph");
+const detectOrphan = require("./features/routing/detect-orphan");
+
+const failFast = errors => {
+  if (errors.errors.length > 0) {
+    throw new Error(JSON.stringify(errors, null, 2));
+  }
+};
+
+const validatePage = (
+  page,
+  ctx,
+  errors = {
+    errors: []
+  },
+  graph
```

Comment:
Seems like you can put a function call in the default so this could be `graph = buildRoutingGraph(ctx)` might work.

Reply:
Oh, really? Fantastic. I've made that change.